.droneci/config.yml

```diff
@@ -4,30 +4,56 @@ name: default
 steps:
 - name: build
   image: ivigns/nice-ubuntu:0.0.1
+  volumes:
+  - name: cache
+    path: /tmp/cache
   commands:
   - git submodule sync
   - git submodule update --init --recursive
-  - git submodule update --recursive
-  - cd /drone/src/sdsl-lite
-  - bash uninstall.sh /usr
+  - git submodule foreach --recursive git pull origin master
   - cd /tmp
-  - rm -rf sdsl-lite-build
-  - rm -rf libdivsufsort-build
   - mkdir sdsl-lite-build
   - mkdir libdivsufsort-build
   - export CC=clang
   - export CXX=clang++
-  - export CFLAGS="-O3 -pipe -fPIC"
-  - export CXXFLAGS="-O3 -pipe -fcolor-diagnostics -static-libgcc -fPIC -lpthread -Wl,-Bstatic -Wl,-Bdynamic"
```

Comment:
If you're not using static libs the wheels wouldn't work if libgcc and some other libs don't match with ones installed on target system. As a test you can try installing resulting wheel to alpine based system or to old fedora or debian.

Reply:
So add simple test please (at least install and import)